source/types/retry.ts

```diff
@@ -33,4 +33,19 @@ export interface RetryOptions {
 	@default Infinity
 	*/
 	maxRetryAfter?: number;
+
+	/**
+	The upper limit of the delay per retry in milliseconds.
+
+	By default, the delay is calculated in the following way:
+
+	```
+	0.3 * (2 ** (attemptCount - 1)) * 1000
+	```
+
+	The delay increases exponentially.
+
+	In order to prevent this, you can set this value to a fixed value, such as 1000.
+	*/
+	backoffLimit?: number;
```

Comment:
The docs for TS and readme should be in sync as much as possible. It makes it hard to maintain if they differ significantly.